main.py

```
@@ -106,10 +106,16 @@ def main():
         safe_print(f"   {tool_icons[tool]} {tool.title()} - Google {tool.title()} API integration")
     print()
 
+    # Check authentication mode
+    access_token = os.getenv('GOOGLE_OAUTH_ACCESS_TOKEN')
+    auth_method = "Access Token (GOOGLE_OAUTH_ACCESS_TOKEN)" if access_token else "OAuth 2.0 with PKCE"
+    
     safe_print(f"📊 Configuration Summary:")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)